turborepo-tests/integration/tests/command-ls.t

```diff
@@ -0,0 +1,37 @@
+Setup
+  $ . ${TESTDIR}/../../helpers/setup_integration_test.sh
+
+Run info
+  $ ${TURBO} ls
+   WARNING  ls command is experimental and may change in the future
+  3 packages
+  
+    another packages/another
+    my-app apps/my-app
+    util packages/util
+
+Run info with filter
+  $ ${TURBO} ls -F my-app...
```

Comment:
Playing around with it, I immediately found myself wanting `--recursive` flag. Does `...` recurse all deps or just first level?